packages/flame/lib/src/components/component.dart

```diff
@@ -130,8 +131,10 @@ class Component with Loadable {
 
   void render(Canvas canvas) {}
 
-  void renderTree(Canvas canvas) {
-    render(canvas);
+  void renderTree(Canvas canvas, {bool callRender = true}) {
```

Comment:
What if we call callRender to skipOwnRender, to make it clear that the flag is about to the component own render and not it's children?